mysql-test/suite/innodb_fts/t/percona_ft_ignore_ngram_check.test

```diff
@@ -0,0 +1,77 @@
+--source include/have_innodb.inc
+
+#
+# bp:innodb-fts-ngram-ignore-stopword-list
+# "A new InnoDB variable to control whether InnoDB FTS NGRAM parser should ignore stopword list"
+#
+# lp:1679135 "Ignore INNODB_FT_DEFAULT_STOPWORD for ngram indexes"
+# Bug #84420 "stopwords and ngram indexes"
+#
+
+SET innodb_ft_ignore_ngram_check = 0;
```

Comment:
We need to rename this variable: the current name "innodb_ft_ignore_ngram_check" is not clear at all, does not mention anything about stopwords ("what ngram checks are ignored?")

innodb_ft_ngram_use_stopwords? (default TRUE)

Reply:
Agree that the name should include `stopwords` part. However, don't like the idea of reversing the value. My reasoning are the following:
If the default value is FALSE and we need this option to be enabled via command line, we can start `mysqld --innodb_ft_ignore_stopwords=ON` or just `mysqld --innodb_ft_ignore_stopwords`
whereas when the default is TRUE, the only way to change it at startup is
`mysqld --innodb_ft_ngram_use_stopwords=OFF` because the short form `mysqld --innodb_ft_ngram_use_stopwords` will mean no changes.